client/app/initialize.coffee

```diff
@@ -11,27 +11,44 @@ Logging
 
 Send client side errors to server.
 ###
-window.onerror = (msg, url, line, col, error) ->
-    console.error msg, url, line, col, error, error?.stack
-    exception = error?.toString() or msg
-    if exception isnt window.lastError
+initReporting = ->
+    env     = require('imports').env
+    console = console or window.console or {}
```

Comment:
To be totally honnest, I was inspired by [the raven-js console plugin](https://github.com/getsentry/raven-js/blob/master/plugins/console.js). It's useful when dealing with both node / browser env, but useless in this case… I'll remove it.